nautobot_chatops/dispatchers/slack.py

```diff
@@ -208,16 +213,30 @@ def send_blocks(self, blocks, callback_id=None, modal=False, ephemeral=False, ti
             self.send_exception(slack_error)
 
     @BACKEND_ACTION_SNIPPET.time()
-    def send_snippet(self, text, title=None):
+    def send_snippet(self, text, title=None, ephemeral=None):
         """Send a longer chunk of text as a file snippet."""
+        if ephemeral is None:
+            ephemeral = settings.PLUGINS_CONFIG["nautobot_chatops"]["send_all_messages_private"]
+
         if self.context.get("channel_name") == "directmessage":
             channels = [self.context.get("user_id")]
         else:
             channels = [self.context.get("channel_id")]
         channels = ",".join(channels)
         logger.info("Sending snippet to %s: %s", channels, text)
         try:
-            self.slack_client.files_upload(channels=channels, content=text, title=title)
+            # Check for the length of the file if the setup is meant to be a private message
+            if ephemeral:
+                message_list = self.split_messages(text, 40000)
+                for msg in message_list:
+                    self.slack_client.chat_postEphemeral(
+                        channel=self.context.get("channel_id"),
+                        user=self.context.get("user_id"),
+                        text="test",
```

Comment:
Probably unneeded?

Reply:
I think this is needed. This is how it is sent ephemerally.

Reply:
IIRC I was referring specifically to `text="test"` as being unneeded.